textattack/attack_recipes/textbugger_li_2018.py

```diff
@@ -0,0 +1,72 @@
+from textattack.search_methods import GreedyWordSwapWIR
+from textattack.shared.attack import Attack
+from textattack.constraints.grammaticality import PartOfSpeech
+from textattack.constraints.pre_transformation import RepeatModification, StopwordModification
+from textattack.constraints.semantics.sentence_encoders import UniversalSentenceEncoder
+from textattack.transformations import CompositeTransformation, \
+    WordSwapRandomCharacterInsertion, \
+    WordSwapRandomCharacterDeletion, WordSwapNeighboringCharacterSwap, \
+    WordSwapHomoglyphSwap, WordSwapEmbedding
+from textattack.goal_functions import UntargetedClassification
+
+def TextBuggerLi2018(model):
+    """
+    Li, J., Ji, S., Du, T., Li, B., and Wang, T. (2018).
+    
+    TextBugger: Generating Adversarial Text Against Real-world Applications.
+ 
+    ArXiv, abs/1812.05271.
+    """
+    #
+    #  we propose five bug generation methods for TEXTBUGGER:
+    #
+    transformation = CompositeTransformation([
+        # (1) Insert: Insert a space into the word.
+        # Generally, words are segmented by spaces in English. Therefore,
+        # we can deceive classifiers by inserting spaces into words.
+        WordSwapRandomCharacterInsertion(random_one=True, letters_to_insert=' ', skip_first_char=True, skip_last_char=True),
```

Comment:
I still think we should make a separate transformation for splitting up words. It's not a word swap. If you prefer it this way we should discuss it.